jib-core/src/main/java/com/google/cloud/tools/jib/builder/PushContainerConfigurationStep.java

```diff
@@ -20,69 +20,69 @@
 import com.google.cloud.tools.jib.blob.Blob;
 import com.google.cloud.tools.jib.blob.BlobDescriptor;
 import com.google.cloud.tools.jib.hash.CountingDigestOutputStream;
-import com.google.cloud.tools.jib.http.Authorization;
 import com.google.cloud.tools.jib.image.Image;
 import com.google.cloud.tools.jib.image.LayerPropertyNotFoundException;
 import com.google.cloud.tools.jib.image.json.ImageToJsonTranslator;
 import com.google.cloud.tools.jib.registry.RegistryClient;
 import com.google.cloud.tools.jib.registry.RegistryException;
-import com.google.common.collect.ImmutableList;
 import com.google.common.io.ByteStreams;
 import com.google.common.util.concurrent.Futures;
 import com.google.common.util.concurrent.ListenableFuture;
 import com.google.common.util.concurrent.ListeningExecutorService;
 import java.io.IOException;
-import java.util.concurrent.Callable;
 import java.util.concurrent.ExecutionException;
 
 /** Pushes the container configuration. */
-class PushContainerConfigurationStep implements Callable<ListenableFuture<BlobDescriptor>> {
+class PushContainerConfigurationStep implements AsyncStep<ListenableFuture<BlobDescriptor>> {
```

Comment:
Should this have a TODO as well for changing `ListenableFuture` to `AsyncStep`? (Like in `BuildImageStep`)

Reply:
Hmm, this makes me think that I should change all these steps to actually be implementing AsyncStep and Callable separately.